src/errors.ts

```diff
@@ -72,6 +72,16 @@ export class EtcdLeaseInvalidError extends Error {
   }
 }
 
+/**
+ * EtcdLockFailedError is thrown when we fail to aquire a lock.
+ */
+export class EtcdLockFailedError extends Error {
+  constructor(message: string) {
+    super(message);
+    Object.setPrototypeOf(this, EtcdLockFailedError.prototype);
```

Comment:
Hm, I was still have some issues with the ts-node-based unit tests. I'll see if there's an option to get it to target ES6.